app/models/katello/sync_plan.rb

```diff
@@ -174,5 +174,25 @@ def rec_logic_changed?
     def cron_status_mismatch?
       self.interval != CUSTOM_CRON && !(self.cron_expression.nil? || self.cron_expression.eql?(''))
     end
+
+    def validate_and_update_products(force_update: false)
```

Comment:
Will we ever need the option to not force update and just log the warning? It looks like in the current implementation, we are always force updating. What was the rationale behind the decision to include that?

Reply:
I was thinking for support purposes and an option to figure out which sync plans have the issue if we needed to..One question I had which is more a PX kind of question is if we should run a script to fix these "broken" sync plans in user environments..

Reply:
Ok, I understand why you included it. That seems reasonable to keep the toggle, then. The support script isn't a bad idea at all; I'm a big fan.